lua/indent-tools/init.lua

```diff
@@ -196,12 +196,24 @@ return {
           vim.keymap.set("n", opts.normal.down, function()
             jump_indent(true)
           end, { desc = "jump down along the indent" })
+          vim.keymap.set("x", opts.normal.down, function()
```

Comment:
How do you feel about extending the line 196 instead?
```lua
vim.keymap.set({ "n", "x", "o" }, opts.normal.down, function()
    jump_indent(true)
end, { desc = "jump down along the indent" })
```

Reply:
Yes, good suggestion, that shortens it quite a bit.

I'll only be able to use it for "n" and "x" though, since the function call for "o" is different.

I've updated the pull request with the changes.

Reply:
It's perfect, thank you!